app/src/main/res/navigation/root_nav_graph.xml

```diff
@@ -133,6 +133,10 @@
         android:id="@+id/action_open_select_wallet"
         app:destination="@id/selectWalletFragment" />
 
+    <action
```

Comment:
If we always use root navigaiton, does it mean that the shown screen will always overlay the minimized browser view?